source/documentation/administration_guide/topics/proc-adding-kubevirt-openshift-as-an-external-provider.adoc

```diff
@@ -36,7 +36,10 @@ NOTE: This capability is known as _OpenShift Virtualization_.
 
 . Click the *Hosts* tab to verify that the {container-platform} status of the worker nodes is up.
 +
-NOTE: Even if they are running, the status of the master nodes is down because they cannot host virtual machines.
+[NOTE]
+====
+Even if they are running, the status of the control plane nodes is down because they cannot host virtual machines.
```

Comment:
I think this line could be made a little clearer. Suggestion:

The status of the control plane nodes is `down`, even if they are running, because they cannot host virtual machines.